### ISSUE TITLE
Set up generate a QR code role on the BA OpenMRS Bahmni version

### ISSUE THREAD

**Issue by msschroeder** [OpenSRP/opensrp-client-chw#264]
- [ ] Set up the generate a QR code role on the BA OpenMRS Bahmni version

**Comment by rkodev** [comment]
@msschroeder @keymane kindly confirm if integration with bahmni is still necessary?

**Comment by msschroeder** [comment]
@rkodev Now it is looking like there will be using GoT-HOMIS instead of OpenBahmni. But OpenBahmni, is still technically part of our contract. @mberg @cagulas What are you thoughts? I think we could use the instance of OpenMRS we set up for the project already.

**Comment by cagulas** [comment]
Yes, we'll use the instance we have set up. OpenMRS/Bahmni will always need to be in the background, since we use it for user mgmt, location syncing, etc.

I would say we should move this issue to the icebox for now, until we have a plan for implementation of QR codes with Jhpiego. From my understanding, there is broad agreement to use QR codes, but haven't discussed any implementation plan for it yet.
